src/components/layout/footer/social-wrapper.tsx

```diff
@@ -33,26 +23,26 @@ type SocialMediaComponentProps = {
     social_accounts: TSocialAccount[]
 }
 
-const SocialWrapperComponent = ({
-    is_career_page = false,
-    fb_url = '',
-    instagram_url = '',
-    twitter_url = '',
-}: SocialWrapperComponentProps) => {
+const SocialWrapperComponent = ({ is_career_page = false }: SocialWrapperComponentProps) => {
+    const fb_url = useSocialMediaUrl('facebook')
+    const instagram_url = useSocialMediaUrl('instagram')
+    const twitter_url = useSocialMediaUrl('twitter')
+    const linkedin_url = useSocialMediaUrl('linkedin')
```

Comment:
Can we have it return objects of social media url, so instead of the above we can do something like: 

```
const { fb_url, instagram_url, twitter_url, linkedin_url } = useSocialMediaUrls()
```

Reply:
Resolved @prince-deriv